bin/MadGraph5_aMCatNLO/cards/ZHhad_ALP_chiral/ZHhad_ALP_chiral_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+import model ALP_chiral_UFO
+define q = u c d s b u~ c~ d~ s~ b~
+define lep+ = e+ mu+ ta+
+define lep- = e- mu- ta-
+define vw = w+ w-
+
+generate g g > z h, (z > q q , h > vw > lep+ lep- vl vl~)
+add process g g > z h, (z > q q , h > a  lep+ lep-)
+add process g g > z h, (z > q q , h > lep+ lep-  lep+ lep-)
+add process g g > z h, (z > q q , h > z > lep+ lep- vl vl~)
+add process g g > z h, (z > q q , h > z > lep+ lep- q q)
+add process g g > z h, (z > q q , h >  q q)
```

Comment:
have you tried with the syntax? i think the parentheses should be removed

Reply:
Hi @menglu21 , yes we have already produced and tested the gridpacks :)